Tella/Data/Networking/Models/UwaziLoginResult.swift

```diff
@@ -7,10 +7,10 @@
 //
 
 import Foundation
-struct UwaziLoginResult: Codable {
-    let success: Bool
-
-    enum CodingKeys: String, CodingKey {
-        case success
-    }
-}
+//struct UwaziLoginResult: Codable {
```

Comment:
Are we going to use this in the future? If not I suggest removing it instead of commenting it

Reply:
The file is removed